apps/judicial-system/backend/src/app/modules/case/models/case.model.ts

```diff
@@ -39,7 +39,7 @@ import {
 
 import { CivilClaimant, Defendant } from '../../defendant'
 import { EventLog } from '../../event-log'
-import { CaseFile } from '../../file'
+import { CaseFile } from '../../file/models/file.model'
```

Comment:
Had some trouble with circular dependencies. I guess its tiime to throw in the towel and eliminate barrel files. Also, its time to refactor/rewrite our modules to minimize co-dependencies.